python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -180,23 +182,23 @@ def get_env():
     """
 
     project_dir: Path
-    target_path: Path
+    output_path: Path
     target: Optional[str]
     manifest_path: Path
     packaged_project_dir: Optional[Path]
     state_path: Optional[Path]
     has_uninstalled_deps: bool
-    manifest_preparer: DbtManifestPreparer
+    preparer: DbtManifestPreparer
```

Comment:
This change makes sense to me given that it's doing more than just generating the manifest.